Marlin/planner.h

```diff
@@ -166,6 +166,11 @@ class Planner {
      * Nominal speed of previous path line segment
      */
     static float previous_nominal_speed;
+	
+	/**
+	 * Limit where 64bit math is necessary for acceleration calculation
+	 */
+	static unsigned long cutoff_32bit;
```

Comment:
Sorry. The diff was to short to see it's in a class. If not pointed with the nose on it, i'm still always thinking in C not C++.